v0.1/core/anatomicalEntity.schema.json

```diff
@@ -14,29 +20,40 @@
       "type": "string",
       "description": "Metadata node identifier.",
       "pattern": "sands/v0.1/core/anatomicalEntity/([A-Za-z0-9_-]+).json"
-    },
-    "nameOfLocation": {
-      "type": "string",
-      "description": "nameOfLocation"
-    },
+    },    
     "laterality": {
       "type": "array",
-      "description": "laterality",
+      "description": "The side of the body or bilateral organ that this anatomical entity relates to.",
       "items": {
         "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM"
-      }
+      },
+      "maxItems": 2
+    },
+    "nameOfLocation": {
+      "type": "string",
+      "description": "The name of the anatomical entity."
     },
     "parentStructure": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/ANNATOMICAL_ENTITY",
-      "description": "parentStructure"
+      "type": "array",
+      "description": "A parent structure to this anatomical entity.",
+      "items": {
+        "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/ANNATOMICAL_ENTITY"
+      },
+      "maxItems": 1
     },
     "source": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM",
-      "description": "source"
-    },
-    "performedBy": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM",
-      "description": "performedBy"
+      "type": "array",
+      "description": "Terminology the anatomical entity originates from.",
+      "items": {
+        "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/TERMINOLOGY"
+      },
+      "relatedOntology": {
```

Comment:
If the maximum item is 1 you don't have to define an array here. I would keep the json-schemas as simgle as possible. You can directly reference to the corresponding schema ("type": "object", "$ref": "...").
Let me know if you think otherwise.

Reply:
if we allow multiple sources, shouldn't we allow multiple relatedOntologies?

Reply:
In the context of a reference atlas: given that a version has been specified, e.g. version x, there can only be a 1-1 relation. 
In the context of a dataset: it is possible to have multiple sources/relatedOntologies, but do we want or need that?

Reply:
I think I would like to keep it simple for now and only allow 1-to-1 relationships. Why did we decided on multiple sources? Could we for now also simplify this to a 1-to-1 relationship?

Reply:
Yes, we will. I talked to Heidi and we will need to revise the brainAtlas schema based on this discussion as well. In essence, it is about "brain atlas" vs. "brain atlas version(s)" and brainAtlas.schema is a mix of both (based on info from Wiki). 
I'll make this an issue in the schema revision issue tomorrow with some more explanations.